src/test/java/ValidateInputTest.java

```diff
@@ -0,0 +1,58 @@
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class ValidateInputTest {
+
+    @Test
+    public void testValidatesInput() {
```

Comment:
can remove `test` from the name of the method, as the annotation above shows that it is a test